samples/cpp/speech_sample/main.cpp

```diff
@@ -232,6 +252,21 @@ int main(int argc, char* argv[]) {
                 throw std::runtime_error("Cannot open model file " + FLAGS_rg);
             }
             executableNet = core.import_model(streamrq, deviceStr, genericPluginConfig);
+            // loading batch from exported model
+            const auto& imported_inputs = executableNet.inputs();
+            if (std::any_of(imported_inputs.begin(), imported_inputs.end(), [](const ov::Output<const ov::Node>& i) {
+                    return ov::layout::get_layout(i).empty();
+                })) {
+                batchSize = 1;
```

Comment:
Here we need warning that batch size is not found and we use 1 as default.

Reply:
ok, I will add

Reply:
@nkogteva , I added it